ui/app/adapters/secrets-engine-path.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/**
+ * General use adapter to access specified paths on secrets engines
+ * For example /:backend/config is a typical use case for this adapter
+ */
+
+import ApplicationAdapter from 'vault/adapters/application';
+import { encodePath } from 'vault/utils/path-encoding-helpers';
+
+export default class KubernetesConfigAdapter extends ApplicationAdapter {
```

Comment:
This is the generic adapter correct? It seems confusing that KubernetesConfig adapter extends this? Should it be something like `SecretsEngineConfigAdapter extends`

Reply:
Oops missed changing that when I copied it over. Thanks!